docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -207,6 +207,19 @@ include::partial$proxy-meta-tags.adoc[]
 
 include::partial$dynamic-variables-ui.adoc[]
 
+== Profiles
+
+=== Docker
+The profile can be enabled by adding `web/docker` to `configuration.profiles` property
+
+Default profile configuration
+[source,properties]
+----
+selenium.grid.enabled=true
+selenium.grid.capabilities.platformName=linux
+selenium.grid.capabilities.browserName=${selenium.browser}
+----
```

Comment:
no reason to duplicate config here, it's better to add an example how to combine the profile with another ones and which properties must be set to use local docker containers